ui/src/views/infra/network/providers/AddSrxFirewall.vue

```diff
@@ -25,8 +25,9 @@
         <a-col :md="24" :lg="24">
           <a-form-item :label="$t('label.ip')">
             <a-input
+              :placeholder="apiParams.url.description"
               autoFocus
-              v-decorator="['ip', {
+              v-decorator="['url', {
```

Comment:
@davidjumani If you change it like this, the current handleSubmit() function is still using `values.ip`?

Reply:
Thanks for the catch @utchoang. Pushing a fix